src/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,151 @@
+          <AvatarBadge size={avatarSize} {...topLeftBadgeProps} />
+        </div>
+      );
+    }
+    if (!isNil(topRightBadgeProps)) {
+      badges.push(
+        <div className={cx(bemHelper({ element: "badge" }), bemHelper({ element: "badge", state: "top-right" }))}>
+          <AvatarBadge size={avatarSize} {...topRightBadgeProps} />
+        </div>
+      );
+    }
+    if (!isNil(bottomLeftBadgeProps)) {
+      badges.push(
+        <div className={cx(bemHelper({ element: "badge" }), bemHelper({ element: "badge", state: "bottom-left" }))}>
+          <AvatarBadge size={avatarSize} {...bottomLeftBadgeProps} />
+        </div>
+      );
+    }
+    if (!isNil(bottomRightBadgeProps)) {
+      badges.push(
+        <div className={cx(bemHelper({ element: "badge" }), bemHelper({ element: "badge", state: "bottom-right" }))}>
+          <AvatarBadge size={avatarSize} {...bottomRightBadgeProps} />
+        </div>
+      );
+    }
+
+    return badges.length > 0 ? <div className={cx(bemHelper({ element: "badges" }))}>{badges}</div> : null;
+  }, [topLeftBadgeProps, topRightBadgeProps, bottomLeftBadgeProps, bottomRightBadgeProps]);
+
+  return (
+    <div className={cx(AVATAR_CSS_BASE_CLASS, className)}>
+      <div
+        className={cx(
+          bemHelper({ element: "circle" }),
+          bemHelper({ element: "circle", state: avatarType }),
+          bemHelper({ element: "circle", state: avatarSize }),
+          {
+            [bemHelper({ element: "circle", state: "is-disabled" })]: isDisabled,
+            [bemHelper({ element: "circle", state: "is-square" })]: isSquare
+          }
+        )}
+        aria-hidden={ariaHidden}
+        tabIndex={tabIndex}
+        style={backgroundColorStyle}
+      >
+        {avatarContent}
+      </div>
+      {badgesContainer}
+    </div>
+  );
+};
+Avatar.propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+  backgroundColor: PropTypes.oneOf(elementColorsNames),
+  role: PropTypes.string,
+  ariaLabel: PropTypes.string,
+  size: PropTypes.oneOf(AVATAR_ALLOWED_SIZES),
+  tabIndex: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  ariaHidden: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  isSquare: PropTypes.bool,
+  topLeftBadgeProps: PropTypes.object,
+  topRightBadgeProps: PropTypes.object,
+  bottomLeftBadgeProps: PropTypes.object,
+  bottomRightBadgeProps: PropTypes.object
+};
+
+Avatar.defaultProps = {
+  className: "",
+  backgroundColor: elementColorsNames.CHILI_BLUE,
+  role: undefined,
+  ariaLabel: "",
+  size: AVATAR_SIZES.LARGE,
+  tabIndex: 0,
+  children: undefined,
+  ariaHidden: false,
+  isDisabled: false,
+  isSquare: false,
+  topLeftBadgeProps: undefined,
+  topRightBadgeProps: undefined,
+  bottomLeftBadgeProps: undefined,
+  bottomRightBadgeProps: undefined
+};
+
+Avatar.sizes = AVATAR_SIZES;
+Avatar.colors = elementColorsNames;
```

Comment:
same comment

Reply:
Avatar.sizes is an object and PropTypes.oneOf need to receive an array such as AVATAR_ALLOWED_SIZES. What can I do in this case?